Include/httpClient/httpClient.h

```diff
@@ -398,6 +398,12 @@ STDAPI HCHttpCallRequestEnableGzipCompression(
     _In_ HCCompressionLevel level
 ) noexcept;
 
+// TODO: Add summary
+STDAPI HCHttpCallSetCompressedResponse(
```

Comment:
The client calls this indicate that it wants the response to be compressed correct? If that's the case 'HCHttpCallSetCompressResponse' makes more sense to me.